hat/assets/js/apps/Iaso/domains/completenessStats/hooks/useCompletenessStatsColumns.tsx

```diff
@@ -77,10 +81,49 @@ export const useCompletenessStatsColumns = (params: any) => {
                 accessor: 'forms_filled_direct',
                 sortable: false,
                 Cell: settings => (
-                    <span>
-                        {settings.row.original.forms_filled_direct ?? '--'}/
-                        {settings.row.original.forms_to_fill_direct ?? '--'}
-                    </span>
+                    <Box>
+                        <div
+                            style={{
+                                // marginTop: '2px', // The margin will align the icon in FF but misalign it in Chrome
+                                display: 'inline-flex',
+                                alignItems: 'center',
+                                verticalAlign: 'middle',
+                            }}
+                        >
+                            {`${
+                                settings.row.original.forms_filled_direct ??
+                                '--'
+                            }/
+                            ${
+                                settings.row.original.forms_to_fill_direct ??
+                                '--'
+                            }`}
+                        </div>
+                        {settings.row.original
+                            .has_multiple_direct_submissions && (
+                            <Tooltip
+                                title={formatMessage(
+                                    MESSAGES.orgUnitHasMultipleSubmissions,
+                                )}
+                            >
+                                <div
```

Comment:
Same here

Reply:
Same as above. The verticalAlign is required here though